helpers/config_parse.py

```diff
@@ -1,26 +1,30 @@
-#!/usr/bin/env python
```

Comment:
Why'd he remove it, I have no idea =) I think it's better to keep it in, it has this obscure debugging purpose =D